From 601ba140845615c616aae411d4a405655df76332 Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Sun, 11 Mar 2012 13:58:03 +0530
Subject: [PATCH 0901/1092] ARM: OMAP2+: ehrpwm: Removes registration of
 EHRPWM.2

Removes registration of EHRPWM instance 2 to support registration using
new API.

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index 0f212d9..ba7604d 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -1030,7 +1030,6 @@ static void uart2_init(int evm_id, int profile)
 static void haptics_init(int evm_id, int profile)
 {
 	setup_pin_mux(haptics_pin_mux);
-	register_ehrpwm(HAPTICS_MAX_FREQ);
 	return;
 }
 
-- 
1.7.11.2

